scapy/as_resolvers.py

```diff
@@ -91,23 +91,31 @@ def resolve(self, *ips):
             asn,ip,desc = map(str.strip, l.split("|"))
             if asn == "NA":
                 continue
-            asn = int(asn)
-            ASNlist.append((ip,asn,desc))
+            asn = "AS" + str(int(asn))
```

Comment:
Why do you change this? The API, for now, is that an integer should be returned, or the string "NA".